pkgs/applications/misc/keepassx/2.0-http.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, libgcrypt, qt5, zlib, libmicrohttpd, libXtst }:
+
+stdenv.mkDerivation rec {
+  name = "keepassx2-http-git-${version}";
+  version = "2.0.2-2016-05-27"; 
```

Comment:
Again, I think you should replace _git_ with _unstable_ and drop the _2.0.2_ ( https://nixos.org/releases/nixpkgs/nixpkgs-16.09pre88945.5120af0/manual/#sec-package-naming ) :

```
name = "keepassx2-http-unstable-${version}";
version = "2016-05-27";
```
Otherwise, the parse will consider the package name "keepassx2-http-git-2.0.2" and the version "2016-05-27".

The git -> unstable part is a new convention \ rule so it's not documented in the older manual.